packages/server/lib/cloud/protocol.ts

```diff
@@ -20,6 +20,8 @@ const DELETE_DB = !process.env.CYPRESS_LOCAL_PROTOCOL_PATH
 // Timeout for upload
 const TWO_MINUTES = 120000
 
+const DB_SIZE_LIMIT = 5000000000
```

Comment:
Should this be:

```suggestion
const DB_SIZE_LIMIT = 5 * 1024 * 1024 * 1024
```

Reply:
I assumed s3 limits were 5 GB rather than 5 GiB, so used base 10

Reply:
I think you're right here.

👍🏻